src/components/MessagesList/MessagesList.vue

```diff
@@ -21,11 +21,13 @@
 			:key="`section_${dateTimestamp}`"
 			:ref="`dateGroup-${token}`"
 			:data-date-timestamp="dateTimestamp"
-			:class="{'has-sticky': dateTimestamp === stickyDate}">
+			class="scroller__content"
+			:class="{ 'has-sticky': dateTimestamp === stickyDate }">
 			<li class="messages-group__date">
 				<span class="messages-group__date-text" role="heading" aria-level="3">
 					{{ dateSeparatorLabels[dateTimestamp] }}
 				</span>
+				<span class="messages-group__date-hidden">{{ getHiddenTime(dateTimestamp) }}</span>
```

Comment:
Just a waste of resources I'd say :p. You can set to static time or string if possible as it will always be 12:00 or we use margin.